code/datums/gamemodes/infestation.dm

```diff
@@ -21,6 +21,12 @@
 		T.ChangeTurf(/turf/closed/wall/resin, T.type)
 	for(var/i in GLOB.xeno_resin_door_turfs)
 		new /obj/structure/mineral_door/resin(i)
+	for(var/i in GLOB.xeno_tunnel_spawn_turfs)
+		var/obj/structure/xeno/tunnel/new_tunnel = new /obj/structure/xeno/tunnel(i)
```

Comment:
Add a hivenumber here and on jelly pods in new, it'll not add to the proper list otherwise.
```suggestion
		var/obj/structure/xeno/tunnel/new_tunnel = new /obj/structure/xeno/tunnel(i, XENO_HIVE_NORMAL)
```